src/dfm/cleaning/configs/default_dedupe_config.yaml

```diff
@@ -0,0 +1,30 @@
+defaults:
+  - _self_
```

Comment:
`_self_` normally is at the end of the defaults list, to override the others, if needed.